Automattic/wp-calypso
# Sharing: Update sharing Redux state to recommended action type naming

Related: #2080

This pull request seeks to bring the sharing Redux state in line with current naming recommendations. Specifically, this includes:

- Suffixing the action type verb per [recent guideline decisions](https://github.com/Automattic/wp-calypso/commit/aff948ec88f67dd58ddc052db75ac5199355488b) ([related discussion](https://github.com/Automattic/wp-calypso/pull/1660/files#r47770586))
- Consistency of fetch/request-related actions (`_REQUEST`, `_REQUEST_SUCCESS`, `_REQUEST_FAILURE`)

__Testing instructions:__

It's not expected that this pull request should have an impact on existing behavior, since action types should be exclusively used within the `state` directory. However, confirm that the current usage of sharing continues to behave as expected by repeating the testing instructions from #338.

Also confirm that Mocha tests pass by running `make test` from the project root directory or directly from `client/state/ui` and `client/state/sites`.

/cc @gwwar
